MockDataNaming.md

```diff
@@ -94,6 +106,10 @@ with the fallback base name
 ```
 POST|-login-|169d720631e603967135cfce10d235e94aac22b87500ea09d1be295f5b300dca
 ```
+and an optional wildcard base name
+```
+POST|-login-|*
```

Comment:
OK. Might be worth noting this in the README since it's not 100% clear the wildcards are this coarse-grained.